clustered/integration-test/src/test/java/org/ehcache/clustered/management/ManagementClusterConnectionTest.java

```diff
@@ -166,13 +169,29 @@ public void test_reconnection() throws Exception {
               .count();
 
       if (count == 1) {
-        return;
+        break;
+      } else {
+        Thread.sleep(1_000);
       }
+    }
 
-      Thread.sleep(1_000);
+    if (Thread.currentThread().isInterrupted()) {
+      fail();
+    } else {
```

Comment:
No need for a `else`1 here since it's a fail. And it should be an assert. `assertThat(Thread.currentThread().isInterrupted()).isFalse()`